gradle4/plan.sh

```diff
@@ -0,0 +1,27 @@
+source "$(dirname "${BASH_SOURCE[0]}")/../gradle/plan.sh"
+
+pkg_name=gradle
+pkg_origin=core
+pkg_version=4.10.3
+pkg_source=https://services.gradle.org/distributions/${pkg_name}-${pkg_version}-bin.zip
+pkg_shasum=8626cbf206b4e201ade7b87779090690447054bc93f052954c78480fa6ed186e
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="A powerful build system for the JVM"
+pkg_upstream_url=http://gradle.org
+pkg_license=('Apache-2.0')
+pkg_bin_dirs=(bin)
+pkg_lib_dirs=(lib)
+pkg_build_deps=(
```

Comment:
Lots of these are no longer necessary with the inclusion of the `gradle/plan.sh`. this plan should only include what needs to be overridden to avoid duplication.